src/image-upload/image.service.ts

```diff
@@ -5,6 +5,23 @@ import { Observable } from 'rxjs/Observable';
 @Injectable()
 export class ImageService {
   constructor(private http: Http) {
+     /*
+    Here it is important to note that the Internet explorer browser must enter the catch, execute inside can perform
+     file = new File([fileBlob], fileUrl); 
+    */
+    const YaunlaideFile = (<any>window).File;
```

Comment:
I don't know what `YaunlaideFile` means, perhaps something like `fileConstructor` or `originalFileConstructor` would be more appropriate.